8vim/src/main/java/inc/flide/vim8/ui/AboutUsActivity.java

```diff
@@ -5,20 +5,25 @@
 import android.net.Uri;
 import android.os.Bundle;
 import android.view.MotionEvent;
+import android.view.View;
 import android.widget.ImageView;
 
 import androidx.appcompat.app.AppCompatActivity;
+import androidx.cardview.widget.CardView;
 import androidx.constraintlayout.widget.ConstraintLayout;
 
 import inc.flide.vim8.R;
 
 public class AboutUsActivity extends AppCompatActivity {
 
     private ConstraintLayout github;
-    private ConstraintLayout twitter;
+    private ConstraintLayout matrix;
     private ConstraintLayout website;
+    private ConstraintLayout twitter;
     private ConstraintLayout googlePlayStore;
 
+    CardView cardView_back_arrow;
```

Comment:
What is this?
There is no need for this. Always remove unused variables before final commit.